packages/test/test-end-to-end-tests/src/test/sharedInterval.spec.ts

```diff
@@ -571,332 +571,336 @@ describeCompat("SharedInterval", "NoCompat", (getTestObjectProvider, apis) => {
 			}
 		});
 
-		it("Conflicting ops", async () => {
```

Comment:
you should click the gear above and choose hide-whitespace to get a better PR experience here